src/chargemaster_parsers/parsers/uci.py

```diff
@@ -0,0 +1,53 @@
+from . import ChargeMasterEntry, ChargeMasterParser
+import json
+
+class UCIChargeMasterParser(ChargeMasterParser):
+        INSTITUTION_NAME = "UCI"
+        ARTIFACT_URL = "https://www.ucihealth.org/-/media/files/xlsx/patients-visitors/952226406-regentsoftheuniversityofcaliforniaatirvinehospital-standardcharges.json?la=en&hash=0C51B7235E101EB9E6FA133A2D3AC5C7F297EB5C"
```

Comment:
The GET parameters (everything after the ?) don't seem to be needed. Surprisingly, the hash doesn't even match the file hash, so I'm not sure it's needed and may, at some point, be a liability.